lib/UA.js

```diff
@@ -887,7 +895,7 @@ module.exports = class UA extends EventEmitter
 
     // Seal the configuration.
     const writable_parameters = [
-      'password', 'realm', 'ha1', 'display_name', 'register'
+      'password', 'realm', 'ha1', 'display_name', 'register', 'authorization_user'
```

Comment:
Can you pleas move 'authorization_user' to the first position for consistency?

Reply:
Sure, I will do it